PROJECT

```diff
@@ -1,11 +1,7 @@
-projectName: opentelemetry-operator
```

Comment:
Shouldn't the projectName be kept?

Reply:
I thought the operator-sdk doesn't like it anymore.  However, looking at the source code, it looks like kubebuilder v2 does not have projectName defined but v3 does. I think we were already using v3 schema but version number was set to v2? I'll dig deeper into this and try to figure it out.

--------
Error we get if we keep projectName:
```

FATA[0001] unable to load the configuration: unable to unmarshal config at "PROJECT": error unmarshalling project configuration: error unmarshaling JSON: while decoding JSON: json: unknown field "projectName"
make: *** [bundle] Error 1
```

Reply:
I'm not sure how it was working before but "v2" projects cannot have `projectName`, `layout` or `plugins` keys in the PROJECT file. I updated the file to change the version to "v3" and now it accepts these values. 

I'm not sure what other implications updating to "v3" might have but the generated files look fine to me.

Reply:
I'll compare the generated bundle with previously published one and see if there are any important differences.

Reply:
https://gist.github.com/owais/537b9416ff7564f5f022aa35d50a3da8

Just took a cursory look and looked fine to me. I've to go right now. Will take another detailed look in a while and update here.

Reply:
I think I had problems with webhooks with v3, but let's see if the CI passes. I believe we do have tests exercising the webhooks.